src/scss/main.scss

```diff
@@ -1,8 +1,17 @@
-@import 'o-overlay/main';
+@import "o-buttons/main";
+@import "o-colors/main";
+@import "o-overlay/main";
+@import "o-visual-effects/main";
+
+$system-code: "n-syndication";
```

Comment:
We shouldn't be setting the system code in components, only in systems. If this is needed for demos/testing it should be put into a stylesheet used for that purpose.

Reply:
Added to `./test/main.scss`